src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/SyncCloudContent.targets

```diff
@@ -0,0 +1,31 @@
+<Project ToolsVersion="12.0" DefaultTargets="DownloadBlobsFromAzureTargets" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask TaskName="DownloadFromAzure" AssemblyFile="$(ToolsDir)net45/Microsoft.DotNet.Build.CloudTestTasks.dll"/>
```

Comment:
No need to be done here, the projects that are consuming this have already imported their dir.props, or else they would not be able to use buildtools at all :smile: